app/models/card.rb

```diff
@@ -1,5 +1,7 @@
 class Card < ActiveRecord::Base
   belongs_to :user
+  has_attached_file :picture, styles: {medium: '360x360'}, default_url: "/images/:style/missing.png"
```

Comment:
Line is too long. [100/80]<br>Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.